lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/DriverStatusManager.java

```diff
@@ -152,10 +114,12 @@ public synchronized void onError(final Throwable exception) {
 
     LOG.entering(CLASS_NAME, "onError", exception);
 
-    if (this.isShuttingDownOrFailing()) {
+    if (this.isClosing()) {
       LOG.log(Level.WARNING, "Received an exception while already in shutdown.", exception);
     } else {
+
```

Comment:
I can't find which coding style requires empty line before first statement of the block.